core-implementation/core.ts

```diff
@@ -1,23 +1,27 @@
-import grpc from "@grpc/grpc-js";
-import { addService, loadProtoService } from "../proto.js";
-import { hello } from "./hello/hello.js";
-import { recordTransaction } from "./record_transaction/record_transaction.js";
-import { searchTransactionByUser } from "./search_transaction_by_user/search_transaction_by_user.js";
-import { getAccountBalance } from "./get_account_balance/get_account_balance.js";
-
-export default function addModule(server: grpc.Server) {
-	addService(
-		server,
-		loadProtoService("proto/core/core.proto", "core", "IndexLedger"),
-		{
-			RecordTransaction: recordTransaction,
-			SearchTransactionsByUser: searchTransactionByUser,
-			getAccountBalance: getAccountBalance,
-		},
-	);
-	addService(
-		server,
-		loadProtoService("proto/hello.proto", "helloworld", "Greeter"),
-		{ SayHello: hello },
-	);
-}
```

Comment:
your git is configured to use windows style line endings, please switch to unix style line endings:
https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings

(thats why the diff shows the entire file deleted and recreated)